src/content/docs/journey-performance/slo-tutorial.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: 'Create front-end performance benchmarks with service levels'
+metaDescription: 'Learn how to create and define performance benchmarks with servic levels.'
+---
+
+import sonicSliUiOverview from 'images/sonic_screenshot-full_sli-ui-overview.png'
+
+## Objectives
+
+By the end of this tutorial, you will:
+
+* Understand service level indicators (SLIs) and service level obectives (SLOs).
+* Have created and defined SLIs and SLOs for your front-end experience.
+* Have alerts set up to warn of any SLI or SLO breaches so you can react before they impact your users.
+
+## Intro 
+
+Service levels describe services provided to users within a given period of time, in measurable terms. Service level objectives (SLOs) are the goals set for the availability expected out of a system. Service level indicators (SLIs) are the key measurements and metrics to determine the availability of a system. 
```

Comment:
Maybe something to make this more digestible right away, like "Let's look at the difference between service levels, SLOs and SLIs" (just an example). I think some kind of attention grabber could be good to start with as a value prop.